File: code_templates/actions/node/addCuePoint.js
var cuePoint = new Kaltura.objects.KalturaAdCuePoint();
cuePoint.sourceUrl = req.body.sourceUrl;
cuePoint.startTime = req.body.startTime;
cuePoint.entryId = <%- Lucy.code.variable('answers.entryIdEqual') %>;
client.cuePoint.add(function(result) {
  res.json(result);
}, cuePoint);

File: code_templates/actions/ruby/addCuePoint.rb
cuePoint = KalturaAdCuePoint.new();
cuePoint.source_url = params[:sourceUrl]
cuePoint.start_time = params[:startTime]
cuePoint.entry_id = <%- Lucy.code.variable('answers.entryIdEqual') %>
result = @@client.cue_point_service.add(cuePoint)
render json: result
